### PR TITLE
#52972

### PR DESCRIPTION
**Pull request by IoanaRoxanaPoroch**
Commit for mui library, font and theme.

### CODE REVIEW

File: src/UI/button/CustomButton.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
+import Button from '@mui/material/Button';
+
+import { btnStyles } from './CustomButtonStyles';
+
+
```
Comment (dianamiraute) [inline src/UI/button/CustomButton.js:5]: Pay attention at row spacing & unnecessary console logs

File: src/pages/listingPage/ListingPage.js [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+import PhotoGalery from "components/photoGallery/PhotoGallery";
+import { CustomButton } from "UI/button/CustomButton";
+import { Details, shareBtn } from "./ListingPageStyles";
+import ListingPageStyles from "./ListingPageStyles";
+
+export const ListingPage = ({ title }) => {
+  return (
+    <>
+      <div
+        className="custom-div"
```
Comment (dianamiraute) [inline src/pages/listingPage/ListingPage.js:12]: Maybe you can find a better name for class

File: src/UI/button/CustomButton.js [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+export const CustomButton = ({variant, startIcon, sx, children})=>{
+    console.log("zzz", btnStyles);
+
+    console.log("props", startIcon, variant, children);
+
+   
+
+    return <Button variant={variant} startIcon={startIcon} sx={{...sx, ...btnStyles}}>{children}</Button>
+}
```
Comment (BiancaDianaAlexiuc) [inline src/UI/button/CustomButton.js:15]: I suggest leaving a new line at the end of each file as a best practice

File: src/UI/button/CustomButtonStyles.js [hunk 4]
```diff
@@ -0,0 +1,7 @@
+export const btnStyles = {
+    textTransform:'none',
+    color:'primary',
+    fontSize:'14px',
+    fontWeight:'600',
+
+}
```
Comment (BiancaDianaAlexiuc) [inline src/UI/button/CustomButtonStyles.js:7]: The same here

File: jsconfig.json [hunk 5]
```diff
@@ -0,0 +1,5 @@
+{
+    "compilerOptions": {
+      "baseUrl": "src"
+    }
+  }
```
Comment (BiancaDianaAlexiuc) [inline jsconfig.json:5]: I suggest leaving a new line at the end of each file as a best practice

File: src/mockData/photoGalleryData.js [hunk 6]
```diff
@@ -0,0 +1,27 @@
...
+    },
+    {
+      img: image1,
+      title: "Property picture",
+    },
+    {
+      img: image1,
+      title: "Property picture",
+    },
+  ];
```
Comment (BiancaDianaAlexiuc) [inline src/mockData/photoGalleryData.js:27]: And here add a new line

File: src/components/photoGallery/PhotoGallery.js [hunk 7]
```diff
@@ -0,0 +1,52 @@
...
+import { itemData } from "mockData/photoGalleryData";
+import { CustomButton } from "UI/button/CustomButton";
+
+const listImageStyles = {
+  width: "555px",
+  borderRadius: "12px",
+  margin: "auto",
+};
+
+function srcset(image, size, rows = 1, cols = 1) {
```
Comment (BiancaDianaAlexiuc) [inline src/components/photoGallery/PhotoGallery.js:14]: Maybe you can change the name to  setSrc to keep a consistent naming convention for functions, and it's considered a best practice to use the verb first

File: src/themes/themes.js [hunk 8]
```diff
@@ -0,0 +1,8 @@
+import { createTheme } from "@mui/material";
+
+export const appTheme = createTheme({
+    pallete:{
+        primary:'#0356E8',
+        secondary:'#98A2B3'
+    }
+})
```
Comment (BiancaDianaAlexiuc) [inline src/themes/themes.js:8]: Please add a new line

**Review by BiancaDianaAlexiuc — CHANGES_REQUESTED** [review]

**Pull request merged**